primitives/src/runtime_api.rs

```diff
@@ -92,8 +92,12 @@
 //! Let's introduce two types of `ParachainHost` API implementation:
 //! * stable - used on stable production networks like Polkadot and Kusama. There is only one
 //!   stable API at a single point in time.
-//! * staging - used on test networks like Westend or Rococo. Depending on the development needs
-//!   there can be zero, one or multiple staging APIs.
+//! * staging - methods that are ready for production, but will be released on Rococo first. We can
```

Comment:
This seems to defeat the original purpose of the staging methods.

Reply:
One major goal I remember is that we can keep collecting stuff, in order to not bump versions all the time. This goal is maintained.

Reply:
If we have separate implementation for versi we will be able to keep the initial intent. But it will introduce a lot of duplicated code I'm afraid.